test/LibraryManager.Test/Providers/Unpkg/UnpkgCatalogTest.cs

```diff
@@ -129,42 +121,69 @@ public async Task GetLibraryCompletionSetAsync_Versions()
         [TestMethod]
         public async Task GetLatestVersion_LatestExist()
         {
-            CancellationToken token = CancellationToken.None;
-            const string libraryId = "bootstrap@3.3.0";
-            string[] existing = libraryId.Split('@');
-            string result = await _catalog.GetLatestVersion(existing[0], false, token);
-
-            // It can return null value.
-            if (result != null)
-            {
-                Assert.AreNotEqual(existing[1], result);
-            }
+            const string libraryName = "fakeLibrary";
+            Mocks.WebRequestHandler handler = new Mocks.WebRequestHandler().SetupVersions("fakeLibrary");
+            UnpkgCatalog sut = SetupCatalog(handler);
+
+            string result = await sut.GetLatestVersion(libraryName, false, CancellationToken.None);
+
+            Assert.AreEqual("1.0.0", result);
         }
 
         [TestMethod]
+        [Ignore] // TODO: GetLatestVersion currently doesn't distinguish stable and pre-release versions
         public async Task GetLatestVersion_PreRelease()
         {
-            CancellationToken token = CancellationToken.None;
-            const string libraryId = "bootstrap@3.3.0";
-            string[] existing = libraryId.Split('@');
-            string result = await _catalog.GetLatestVersion(existing[0], true, token);
-
-            // It can return null value.
-            if (result != null)
-            {
-                Assert.AreNotEqual(existing[1], result);
-            }
+            const string libraryName = "fakeLibrary";
```

Comment:
const in a method seems a bit odd, is it really necessary?

Reply:
Necessary?  Probably not.  But it doesn't hurt to make it immutable.